README.md

```diff
@@ -15,7 +15,7 @@ https://github.com/scylladb/scylla/wiki/Using-CCM
 
 Scylla usage examples:
 ---------------------------------------------------------------------
-### To create a 3-node Scylla cluster do:
+### Create a 3-node Scylla cluster:
```

Comment:
Nitpick: I would use the gerund ("creating") instead of the imperative ("create").
But it's just a nitpick, the "create" is also readable - it just sounds like a command, something you must do for some reason.

Reply:
Okay changed to `Creating`.